record/experimental.go

```diff
@@ -0,0 +1,6 @@
+package record
+
+const (
+	// KeySend is the custom record identifier for key send preimages.
+	KeySend uint64 = 5482373484
```

Comment:
nit: should be `KeySendType` to match everything else

Reply:
fixed